src/OrchardCore.Modules/OrchardCore.ContentFields/GraphQL/Startup.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace OrchardCore.ContentFields.GraphQL
 {
-    [RequireFeatures("OrchardCore.Apis.GraphQL")]
+    [RequireFeatures("OrchardCore.Apis.GraphQL","OrchardCore.Users")]
```

Comment:
I don't think this should depend on OrchardCore.Users, there is a dedicated `UserPickerStartup` class.

Reply:
Indeed, there is no need to depend on OrchardCore